### PR TITLE
fix: copy the sourcemap files

### LINKED ISSUE #10997: CSS unnecessary sourcemap declaration
#### How to reproduce this bug:

1. Download latest release from https://github.com/zurb/foundation-sites/archive/v6.4.3.zip
2. Go to dist/css directory
3. Verify that foundation.css last line has a sourcemap declaration, but sourcemap file doesn't exist.

#### What should happen:
It depends. From my point of view the Github release is used by other packages, like webjars, so it would be better if the result CSS has no sourcemap ? I'm not a CSS expert, so I might be wrong.

#### What happened instead:
CSS file has a sourcemap declaration, which results in a 404 error in browser.

#### Browser(s) and Device(s) tested on:
Firefox, Linux

#### Foundation Version(s) you are using:
6.4.3

### PR DESCRIPTION
**Pull request by DanielRuf**
This copies the sourcemap files for the unminified development files.

Closes #10997

### CODE REVIEW

File: gulp/tasks/deploy.js [hunk 1]
```diff
@@ -58,6 +59,9 @@ gulp.task('deploy:dist', ['sass:foundation', 'javascript:foundation'], function(
       .pipe(rename({ suffix: '.min' }))
       .pipe(gulp.dest('./dist/css'))
     .pipe(cssFilter.restore)
+    .pipe(sourcemapFilter)
+    .pipe(gulp.dest('./dist/css'))
```
Comment (ncoden) [inline gulp/tasks/deploy.js]: Missing tabulation here.

**Review by ncoden — APPROVED** [review]
I made some research and there seems to be no problem with publishing sourcemap to production. So LGTM 👍 .

File: gulp/tasks/deploy.js (hunk 1 above)
Comment (DanielRuf) [reply gulp/tasks/deploy.js]: Good catch, will change it tomorrow morning if this is ok.

Comment (DanielRuf) [reply gulp/tasks/deploy.js]: Changed.

**Pull request merged**